### PR TITLE
Adding ignore option to jws-qe/interop-ocp-ci repository

### PR DESCRIPTION
**Pull request by aviavraham**
Adding ignore option to jws-qe/interop-ocp-ci repository

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -2409,7 +2409,7 @@ periodics:
       - --ignore=redhat-openshift-ecosystem/preflight
       - --ignore=insights-qe/iqe-cost-management-plugin
       - --ignore=ansible/tower-qa
-      - --ignore=rh-openjdk/jdkContainerOcpTests
+      - --ignore=rh-openjdk/jdkContainerOcpTests --ignore=jws-qe/interop-ocp-ci
```
Comment (smg247) [inline ci-operator/jobs/infra-periodics.yaml:2412]: please follow the convention and put new options on their own lines

Comment (aviavraham) [reply ci-operator/jobs/infra-periodics.yaml]: Thanks, @smg247, for your review. I fixed the typo in the YAML file.

Comment (smg247) [reply ci-operator/jobs/infra-periodics.yaml]: I meant all of the lines, not just this occurrence...

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ presubmits:
         - --ignore=redhat-openshift-ecosystem/cvp
         - --ignore=insights-qe/iqe-cost-management-plugin
         - --ignore=ansible/tower-qa
-        - --ignore=rh-openjdk/jdkContainerOcpTests
+        - --ignore=rh-openjdk/jdkContainerOcpTests --ignore=jws-qe/interop-ocp-ci
```
Comment (smg247) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:42]:
```suggestion
        - --ignore=rh-openjdk/jdkContainerOcpTests
        - --ignore=jws-qe/interop-ocp-ci
```

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 3]
```diff
@@ -101,7 +101,7 @@ presubmits:
...
```
Comment (smg247) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml:104]:
```suggestion
        - --ignore=rh-openjdk/jdkContainerOcpTests 
        - --ignore=jws-qe/interop-ocp-ci
```

**Review by smg247 — COMMENTED** [review]
You are going to want to add the ignore to the `periodic-check-gh-automation` test as well.

**Comment by smg247** [comment]
Still missing one comment:

> You are going to want to add the ignore to the periodic-check-gh-automation test as well.

**Pull request merged**